### ISSUE TITLE
texlivePackages: doc containers still not appearing in binary cache

### ISSUE THREAD

**Issue by xworld21** [NixOS/nixpkgs#367072]
## Describe the bug
The texdoc containers are not available in the binary cache, despite having created `texlivePackages` for that purpose.

## Steps To Reproduce

Build `texlivePackages.amsmath.texdoc`: it will be downloaded from Norbert Preining's personal server.

## Expected behavior

All packages in `texlivePackages` should be built and cached.

## Additional context

Is https://github.com/NixOS/nixpkgs/pull/351919 related to this?
https://github.com/NixOS/nixpkgs/blob/9f36206578abde5ab43e512916a2099b290df88a/pkgs/tools/typesetting/tex/texlive/build-texlive-package.nix#L70-L73

## Notify maintainers

@veprbl, cc @emilazy

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by emilazy** [comment]
Seems potentially related? The idea is that the full combined packages contain all of these things so Hydra will end up building them even if they don’t have separate jobs (which are a technical bottleneck). Are these packages not contained in the combined packages? I think @vcunat may understand this setup better than me.

**Comment by vcunat** [comment]
I don't recall much anymore.  We really do want to avoid Hydra jobs for each individual texlive package.  The heaps of "cheap" jobs are a problem even without adding these.

**Comment by xworld21** [comment]
> Are these packages not contained in the combined packages?

No, the texdoc packages add some 4GB, they are excluded by default.

> We really do want to avoid Hydra jobs for each individual texlive package.

I see. In that case, should we add a `texliveFullDocs` package for the sole purpose of building the texdoc containers? Paired up with the way overdue update that enables `.overrideAttrs { withDocs = true; }` and allows us to deprecate `texlive.combine`.

**Comment by emilazy** [comment]
That seems like a reasonable way forward to me.

**Comment by xworld21** [comment]
All texdoc containers are now in cache, thanks to the new `texliveFullWithDocs`.
